app/controllers/favorites_controller.rb

```diff
@@ -1,6 +1,7 @@
 class FavoritesController < ApplicationController
   def index
-    @favorites = favorite.contents
+    @applied_pets = favorite.pets_w_apps
+    @unapplied_pets = favorite.pets_wo_apps
```

Comment:
Two crucial instance variables needed for the method. Well done on implementation here.